src/sql/workbench/browser/modelComponents/componentBase.ts

```diff
@@ -308,7 +308,7 @@ export abstract class ContainerBase<T, TPropertyBag extends azdata.ComponentProp
 			if (event.eventType === ComponentEventType.validityChanged) {
 				this.validate();
 			}
-		}), false);
+		}), true);
```

Comment:
Add a comment for the name of this parameter that is being set to true.

Reply:
Don't really see that being useful for a single parameter like this - that's just extra noise and with intellisense isn't really doing a whole lot (plus if you change the name of the param or something then you have to remember to go update that comment which is annoying)